src/main/scala/com/ing/wbaa/rokku/sts/service/UserTokenDbService.scala

```diff
@@ -13,9 +13,9 @@ trait UserTokenDbService extends LazyLogging with TokenGeneration {
 
   implicit protected[this] def executionContext: ExecutionContext
 
-  protected[this] def getAwsCredential(userName: UserName): Future[Option[AwsCredential]]
+  protected[this] def getAwsCredential(userName: UserName): Future[(Option[AwsCredential], Boolean)]
```

Comment:
the method name does not match what is returned 
```suggestion
  protected[this] def getAwsCredentialAndStatus(userName: UserName): Future[(Option[AwsCredential], Boolean)]
```